trunk/NDHMS/Routing/module_NWM_io.F

```diff
@@ -305,6 +309,12 @@ subroutine output_chrt_NWM(domainId)
       allocate(velocityLocal(RT_DOMAIN(domainId)%NLINKS))
       strFlowLocal = RT_DOMAIN(domainId)%QLINK(:,1)
       velocityLocal = RT_DOMAIN(domainId)%velocity
+      ! TML: Add qloss allocation and compute variable, only for channel 
+      ! option #2, where qloss is active (muskingum-cunge).
+      if(nlst_rt(domainId)%channel_option .eq. 2) then
+         allocate(qlossLocal(RT_DOMAIN(domainId)%NLINKS))
+         qlossLocal = RT_DOMAIN(domainId)%qloss !TML temp fix to test code
```

Comment:
Only get qloss in NWM configuration. This could be updated later to set to a generic value of no-data value if NWM configuration is not selected.

Reply:
that's fine for now.  I would like to discuss whether we are eventually going to add a hydro.namelist option to turn this feature on/off.  Right now its deactivated when there's no value for ChannK in the Routelink, but that's not very explicit for the user.